x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/execution/search/OrdinalTests.java

```diff
@@ -12,23 +12,26 @@
 @SuppressWarnings("unchecked")
 public class OrdinalTests extends ESTestCase {
 
+    @SuppressWarnings("rawtypes")
```

Comment:
I don't find any value in adding the annotations at method level - it happens for all methods hence why it's defined at the class level.

Reply:
Agreed, it's only one method that doesn't use it, so I'll move it to the class instead.